test/features/data_delete.feature

```diff
@@ -2,13 +2,12 @@ Feature: API endpoint allowing users to remove a record from the Digital Registr
   Request endpoint: DELETE /data/{registryname}/{versionnumber}/{id}/delete
 
   Scenario: The user successfully removes a record from the Digital Registries database
-    Given The user wants to remove record with id=id231Q from the Digital Registries database
-    And The record with id=id231Q does exist in the database
-    When The user triggers an action to delete the database record with with id=id231Q
-    Then Operation to delete record with with id=id231Q finishes successfully
+    Given The user wants to remove the record from the Digital Registries database
+    And The record does exist in the database
```

Comment:
I would use: 
```The record exists in the database```

Reply:
I will leave it because all versions are correct :)